xla/service/gpu/tests/dynamic_update_slice_inplace_bf16.hlo

```diff
@@ -0,0 +1,221 @@
+// RUN: hlo-opt %s --platform=gpu --stage=llvm-before-optimizations --xla_gpu_target_config_filename=%S/../../../tools/hlo_opt/gpu_specs/%{GPU}.txtpb --split-input-file | FileCheck --check-prefixes=CHECK %s
```

Comment:
Maybe instead of opaque "GPU" specifically select the platform you're after (probably need both H100 and A100?)